spring-cloud-aws-parameter-store-config/src/test/java/io/awspring/cloud/paramstore/AwsParamStorePropertiesTest.java

```diff
@@ -78,6 +79,18 @@ void acceptsBackslashAsProfileSeparator() {
 		assertThatNoException().isThrownBy(properties::afterPropertiesSet);
 	}
 
+	@Test
+	void checkExceptionLoggingForPrefix() {
+		try {
+			AwsParamStoreProperties properties = new AwsParamStorePropertiesBuilder().withPrefix("!.").build();
+			properties.afterPropertiesSet();
+		}
+		catch (Exception e) {
+			assertEquals(e.getMessage(),
```

Comment:
here use assertj or if there is a good reason to not use assertj that i don't remember, swap these parameters as "expected" is the first one.